assets/js/base/context/hooks/cart/use-store-cart.ts

```diff
@@ -138,8 +138,7 @@ export const useStoreCart = (
 	const { shouldSelect } = options;
 	const currentResults = useRef();
 
-	// This will keep track of jQuery and DOM events triggered by other blocks
-	// or components and will invalidate the store resolution accordingly.
+	// This will keep track of jQuery and DOM events that invalidates the store resolution accordingly.
```

Comment:
I find the new comment a bit confusing to read. What about something like this?
```suggestion
	// This will keep track of jQuery and DOM events that invalidate the store resolution.
```